app/views/Data/Workspace/WorkTable/index.tsx

```diff
@@ -100,26 +103,48 @@ const workspaceTableQueryDocument = graphql(/* GraphQL */ `
             id
             name
             isAddedToWorkspace
+            columnsCount
+            rowsCount
             dataRows
             dataColumnStats {
                 key
                 label
-                type
-                totalCount
-                uniqueCount
-                stdDeviation
-                minLength
-                naCount
-                min
-                median
+                max
                 mean
                 maxLength
-                max
+                median
+                min
+                minLength
+                naCount
+                stdDeviation
+                totalCount
+                type
+                uniqueCount
             }
         }
     }
 `);
 
+const tableActionMutationDocument = graphql(/* graphql */`
+    mutation changeTableAction(
```

Comment:
mutation name is somewhat confusing here, depicting different meanings. Let's have a proper mutation name.